bugalink-backend/trips/views.py

```diff
@@ -330,3 +336,77 @@ class ReportIssueGetViewSet(viewsets.GenericViewSet, mixins.RetrieveModelMixin):
     @action(detail=True, methods=["get"])
     def get(self, request, *args, **kwargs):
         return self.retrieve(request, *args, **kwargs)
+
+
+class CreateNextWeekTrip(viewsets.GenericViewSet):
```

Comment:
This should be a method inside the main class for trips, not a separate class. Read again the backend development guide we have in case you have doubts